Branch-SDK-TestBed/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,7 @@
         android:allowBackup="true"
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
+        android:theme="@style/AppTheme"
         android:name="io.branch.branchandroiddemo.CustomBranchApp" >
```

Comment:
It'd be awesome to have an activity (or separate page) that shows us the state of the PrefHelper, just read out and display all the contents